src/main/charts/confluence/values.yaml

```diff
@@ -179,6 +182,15 @@ ingress:
   # Optional if Ingress Controller is configured to use one secret for all ingresses
   tlsSecretName:
 
+fluentd:
+  # -- True if the fluentd sidecar should be added to each pod
+  enabled: false
+  # -- The name of the image containing the fluentd sidecar
+  imageName: atlassian/fluentd-elasticsearch:1.0
```

Comment:
Note - this image doesn't actually exist yet. Will need to publish it before we release this.

Reply:
Can't we use https://hub.docker.com/r/fluent/fluentd-kubernetes-daemonset/tags?page=1&ordering=last_updated?

Reply:
Maybe? I'm not really sure what's in it.

Reply:
Worth giving it a go? 
```
docker run -ti --entrypoint=bash fluent/fluentd-kubernetes-daemonset:v1.11.5-debian-elasticsearch7-1.2
root@6479773cb876:/home/fluent#
root@6479773cb876:/home/fluent# gem list | grep "fluent-plugin-elasticsearch"
fluent-plugin-elasticsearch (4.3.3)
```
That's an official repo, and this tag seems to have what we need. Makes things simpler for us.

Reply:
Seems to work! Thanks!